### ISSUE TITLE
can't work when use  postcss-scss

### ISSUE THREAD

**Issue by nyrf** [g45t345rt/esbuild-style-plugin#7]
When i use `postcss-scss` and add inline CSS comments 

```css
body {
  // color: red;
  background: url(images/logo.png);
}
```

postcss.config.js

```js
module.exports = {
  parser: 'postcss-scss',
  plugins: [
    require('postcss-import'),
    require('tailwindcss/nesting'),
    require('tailwindcss'),
    require('autoprefixer')
  ]
}
```

```bash
 ../../node_modules/postcss/lib/input.js:148:15:
      148 │       result = new CssSyntaxError(
          ╵                ^

    at Input.error (/Users/nyrf/import_demo/node_modules/postcss/lib/input.js:148:16)
    at Parser.unknownWord (/Users/nyrf/import_demo/node_modules/postcss/lib/parser.js:540:22)
    at Parser.decl (/Users/nyrf/import_demo/node_modules/postcss/lib/parser.js:219:16)
    at Parser.other (/Users/nyrf/import_demo/node_modules/postcss/lib/parser.js:128:18)
    at Parser.parse (/Users/nyrf/import_demo/node_modules/postcss/lib/parser.js:72:16)
    at parse (/Users/nyrf/import_demo/node_modules/postcss/lib/parse.js:11:12)
    at new LazyResult (/Users/nyrf/import_demo/node_modules/postcss/lib/lazy-result.js:133:16)
    at Processor.process (/Users/nyrf/import_demo/node_modules/postcss/lib/processor.js:28:14)
    at /Users/nyrf/import_demo/node_modules/esbuild-style-plugin/dist/index.js:163:63
    at async callback (/Users/nyrf/import_demo/node_modules/esbuild/lib/main.js:977:28)

  This error came from the "onLoad" callback registered here:

    ../../node_modules/esbuild-style-plugin/dist/index.js:193:10:
      193 │     build.onLoad({ filter: /.*/, namespace: LOAD_STYLE_NAMESPACE }, onStyleLoad(options));
          ╵           ~~~~~~

    at setup (/Users/nyrf/import_demo/node_modules/esbuild-style-plugin/dist/index.js:193:11)
    at async handlePlugins (/Users/nyrf/import_demo/node_modules/esbuild/lib/main.js:881:11)

  The plugin "esbuild-style-plugin" was triggered by this import

    application.js:7:7:
      7 │ import './stylesheets/main.css'
```

**Comment by nyrf** [comment]
I also tested `postcss-strip-inline-comments` and got same error.

**Comment by g45t345rt** [comment]
Should be fixed! Update to `1.4.0` and close the issue if it works.

Use with postcss.config.js
```js
module.exports = {
  parser: require('postcss-scss'),  plugins: [
     require('postcss-strip-inline-comments')
  ]
}
```

Thanks for reporting!

**Comment by nyrf** [comment]
@g45t345rt It works now, thanks for your great work.
